utils/bzip2.xml

```diff
@@ -2,14 +2,25 @@
 <?xml-stylesheet type='text/xsl' href='interface.xsl'?>
 <interface uri="http://repo.roscidus.com/utils/bzip2" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
   <name>Bzip2</name>
-  <summary xml:lang="en">Bzip2: file compressor</summary>
-  <description xml:lang="en">bzip2 is a freely available, patent free, high-quality data compressor. It typically compresses files to within 10% to 15% of the best available techniques (the PPM family of statistical compressors), whilst being around twice as fast at compression and six times faster at decompression. The code is organised as a library, with a programming interface. The bzip2 program itself is a client of the library. You can use the library in your own programs, to directly read and write .bz2 files, or even just to compress data in memory using the bzip2 algorithms. </description>
+  <summary xml:lang="en">A program and library for data compression</summary>
+  <description xml:lang="en">A Windows port of bzip2 and of libbz2.
+
+  GnuWin32 has only a very old version.  MinGW does offer version
+  1.0.6, but it was built in a way that makes the binaries dependent
+  on the DLL version of libgcc.  This means that anyone redistributing
+  these binaries would have to also provide the humongous 75-MB source
+  tarball of GCC, to comply with the GPL license.  I think this is
+  ridiculous, so I made my own port, which is free of this
+  restriction.  While at that, I also fixed a couple of bugs in the
+  existing ports (e.g., try &quot;bunzip2 -c SOMETHING.bz2 &gt; nul&quot;, which is
+  important for measuring decompression speed, if for nothing else).
+</description>
```

Comment:
I think it would be better to keep the original description here, since this feed also provides non-Windows builds of bzip2.

Reply:
done